rdsbroker/config.go

```diff
@@ -11,6 +11,7 @@ type Config struct {
 	BrokerName                   string  `json:"broker_name"`
 	AWSPartition                 string  `json:"aws_partition"`
 	MasterPasswordSeed           string  `json:"master_password_seed"`
+	AWSTagCacheSeconds           uint    `json:"aws_tag_cache_seconds"`
```

Comment:
You should add this to [the config file template](https://github.com/alphagov/paas-rds-broker/blob/main/release/jobs/rds-broker/templates/config/rds-config.json.erb) and [the bosh spec file](https://github.com/alphagov/paas-rds-broker/blob/main/release/jobs/rds-broker/spec) so we can configure it through an ops file in the future.

Reply:
And also add it to the docs